macros/helpers/status_leds.cfg

```diff
@@ -89,62 +91,74 @@ gcode:
 gcode:
     {% set transmit=params.TRANSMIT|default(1) %}
     _SET_LEDS_BY_NAME LEDS="logo" COLOR="off" TRANSMIT={transmit}
-    
+
+
 [gcode_macro SET_NOZZLE_LEDS_ON]
 gcode:
     {% set transmit=params.TRANSMIT|default(1) %}
     _SET_LEDS_BY_NAME LEDS="nozzle" COLOR="on" TRANSMIT={transmit}
 
+
 [gcode_macro SET_NOZZLE_LEDS_OFF]
 gcode:
     {% set transmit=params.TRANSMIT|default(1) %}
     _SET_LEDS_BY_NAME LEDS="nozzle" COLOR="off" TRANSMIT={transmit}
 
+
 [gcode_macro STATUS_OFF]
 gcode:
     SET_LOGO_LEDS_OFF TRANSMIT=0
     SET_NOZZLE_LEDS_OFF
 
+
 [gcode_macro STATUS_READY]
 gcode:
     _SET_LEDS_BY_NAME LEDS="logo" COLOR="standby" TRANSMIT=0
     _SET_LEDS_BY_NAME LEDS="nozzle" COLOR="standby" TRANSMIT=1
 
+
 [gcode_macro STATUS_BUSY]
 gcode:
     _SET_LEDS_BY_NAME LEDS="logo" COLOR="busy" TRANSMIT=0
     SET_NOZZLE_LEDS_ON
 
+
 [gcode_macro STATUS_HEATING]
 gcode:
     _SET_LEDS_BY_NAME LEDS="logo" COLOR="heating" TRANSMIT=0
     _SET_LEDS_BY_NAME LEDS="nozzle" COLOR="heating" TRANSMIT=1
 
+
 [gcode_macro STATUS_LEVELING]
 gcode:
     _SET_LEDS_BY_NAME LEDS="logo" COLOR="leveling" TRANSMIT=0
     SET_NOZZLE_LEDS_ON
 
+
 [gcode_macro STATUS_HOMING]
 gcode:
     _SET_LEDS_BY_NAME LEDS="logo" COLOR="homing" TRANSMIT=0
     SET_NOZZLE_LEDS_ON
 
+
 [gcode_macro STATUS_CLEANING]
 gcode:
     _SET_LEDS_BY_NAME LEDS="logo" COLOR="cleaning" TRANSMIT=0
     SET_NOZZLE_LEDS_ON
 
+
 [gcode_macro STATUS_MESHING]
 gcode:
     _SET_LEDS_BY_NAME LEDS="logo" COLOR="meshing" TRANSMIT=0
     SET_NOZZLE_LEDS_ON
 
+
 [gcode_macro STATUS_CALIBRATING_Z]
 gcode:
     _SET_LEDS_BY_NAME LEDS="logo" COLOR="calibrating_z" TRANSMIT=0
     SET_NOZZLE_LEDS_ON
 
+
```

Comment:
Please keep this file unmodified for the moment. I prefer to do small PRs with focus on features and keep the typo and formating for a later PR if possible.